src/components/Components.js

```diff
@@ -21,20 +21,71 @@ export const RoundImageWithBorder = ({ image, size }) => (
   />
 )
 
+export class IconWithBackground extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      backgroundColor: 'transparent'
+    }
+  }
+
+  unfocus = () => {
+    this.setState({
+      backgroundColor: 'transparent'
+    })
+  }
+
+  focus = () => {
```

Comment:
see above: "Maybe we should add a comment that says that this method is used by parents to change the styling."